docs/pages/web/utilities/defaultlabelprovider.js

```diff
@@ -43,10 +44,256 @@ export default function DocsPage({ generatedDocGen }: {| generatedDocGen: DocGen
+                'component': 'TextField',
+                'prop': 'accessibilityShowPasswordLabel',
+                'label': 'Show password',
+              },
+              {
+                'component': 'HelpButton',
+                'prop': 'tooltipMessage',
+                'label': 'Click to learn more',
+              },
+              {
+                'component': 'Toast',
+                'prop': 'accessibilityDismissButtonLabel',
+                'label': 'Close toast',
+              },
+              {
+                'component': 'Toast',
+                'prop': 'accessibilityIconSuccessLabel',
+                'label': 'Success message',
+              },
+              {
+                'component': 'Toast',
+                'prop': 'accessibilityIconErrorLabel',
+                'label': 'Error message',
+              },
+              {
+                'component': 'Toast',
+                'prop': 'accessibilityProcessingLabel',
+                'label': 'Processing message',
+              },
+              {
+                'component': 'Upsell',
+                'prop': 'accessibilityDismissButtonLabel',
+                'label': 'Dismiss',
+              },
+              {
+                'component': 'Video',
+                'prop': 'accessibilityMaximizeLabel',
+                'label': 'Maximize',
+              },
+              {
+                'component': 'Video',
+                'prop': 'accessibilityMinimizeLabel',
+                'label': 'Minimize',
+              },
+              {
+                'component': 'Video',
+                'prop': 'accessibilityMuteLabel',
+                'label': 'Mute',
+              },
+              {
+                'component': 'Video',
+                'prop': 'accessibilityPauseLabel',
+                'label': 'Pause',
+              },
+              {
+                'component': 'Video',
+                'prop': 'accessibilityPlayLabel',
+                'label': 'Play',
+              },
+              {
+                'component': 'Video',
+                'prop': 'accessibilityProgressLabel',
+                'label': 'Video progress',
+              },
+              {
+                'component': 'Video',
+                'prop': 'accessibilityUnmuteLabel',
+                'label': 'Unmute',
+              },
+              {
+                'component': 'Video',
+                'prop': 'accessibilityHideCaptionsLabel',
+                'label': 'Hide captions',
+              },
+              {
+                'component': 'Video',
+                'prop': 'accessibilityShowCaptionsLabel',
+                'label': 'Show captions',
+              },
+            ].map(({ component, prop, label }) => (
+              <Table.Row key={`${component}-${prop}`}>
+                <Table.Cell>
+                  <Text>{component}</Text>
+                </Table.Cell>
+                <Table.Cell>
+                  <Text>{prop}</Text>
```

Comment:
can we do here <code>{prop}</code>

Reply:
This causes the text color not to be updated in dark mode

Am I missing something?

Reply:
DONE